### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by Sunlis** [aziz/PlainNotes#47]
I use Sublime with a dark theme (similar to [this one](http://colorsublime.com/?q=1337)) and all of the note themes included with this package are quite jarring in comparison.

I would love to see a darker option alongside the current set of neon note themes.

**Comment by codal-ssharma** [comment]
I agree with @Sunlis. We'd love to have a dark theme for PlainNotes.

**Comment by katerlouis** [comment]
This plugin is way to awesome to not have a dark theme.

**Comment by oivva** [comment]
[Boxy Theme](https://github.com/oivva/st-boxy) fully supports this plugin you can try the one of [these color schemes](https://github.com/oivva/st-boxy/tree/dev/schemes) for it.

**Comment by katerlouis** [comment]
Nothing comes even close to the material theme for sublime :)
But maybe I'll just use the color schemes from boxxy as a scheme for plainnotes.
Or I'll write one myself and post it here?
What exactly is different from "usual color schemes"?

**Comment by oivva** [comment]
Material Theme uses pretty similar template for color schemes, but I don't know if it fully supports this package. 

> What exactly is different from "usual color schemes"?

Nothing, this plugin just adds new scopes to markdown those allows [extra markup](https://github.com/aziz/PlainNotes#extra-markup). If you don't need it, you can use markdown highlighting on your notes. Just add Note.sublime-settings to `User` folder. E.g. 

```json
{
	"color_scheme": "Packages/Boxy Theme/schemes/Boxy Yesterday.tmTheme",
	"extensions":
	[
		"md"
	],
	"line_numbers": true,
	"spell_check": true
	"auto_wrap": true
}
```

**Comment by katerlouis** [comment]
Oh, github is so clever! Still I wanna brag about issue 999 😎

**Comment by katerlouis** [comment]
https://github.com/equinusocio/material-theme/issues/999#issuecomment-253643413

> I think that the package developer should be add support for themes, not vice-versa. It should use the current scheme background and the theme md syntax

**Comment by equinusocio** [comment]
@katerlouis thank, i think the package should use the active color scheme and, through a preference, let use the built-in themes.
